src/ArrayManifold.jl

```diff
@@ -130,19 +130,21 @@ function zero_tangent_vector(M::ArrayManifold, x; kwargs...)
     return w
 end
 
-function vector_transport_to(M::ArrayManifold, x, v, y)
+function vector_transport_to(M::ArrayManifold, x, v, y, m)
```

Comment:
I'd change it to `vector_transport_to(M::ArrayManifold, x, v, y, m::AbstractVectorTransportMethod)` (and similarly in other methods). This reduces the chance of accidental ambiguities.

Reply:
I already ran into that at one point and will refactor the remaining ones tomorrow :)

Reply:
Thanks for noticing, yes; I already changed another occurrence, where I actually ran into an accidental ambiguity.